server/executor/functions/functions.go

```diff
@@ -0,0 +1,59 @@
+package functions
+
+import "fmt"
+
+type FunctionInvoker func(args ...FunctionArg) string
```

Comment:
Same for everything. It's a lot easier to read and write `functions.Arg` than `functions.FunctionArg`